examples/classification/main.py

```diff
@@ -139,13 +139,26 @@ def main_worker(current_gpu, config: SampleConfig):
         train_dataset, val_dataset = create_datasets(config)
         train_loader, train_sampler, val_loader, init_loader = create_data_loaders(config, train_dataset, val_dataset)
 
-        def autoq_eval_fn(model, eval_loader):
-            _, top5 = validate(eval_loader, model, criterion, config)
-            return top5
+        def train_steps_fn(loader, model, optimizer, compression_ctrl, train_steps):
+            train_epoch(loader, model, criterion, train_criterion_fn, optimizer, compression_ctrl, 0, config,
+                        train_iters=train_steps, log_training_info=False)
+
+        def validate_fn(model, eval_loader, log=True):
+            top1, top5, loss = validate(eval_loader, model, criterion, config, log)
+            return top1, top5, loss
+
+        execution_params = ExecutionParameters(config.cpu_only,
```

Comment:
@vshampor Please, check the discussed implementation for the distribution initialization mechanism.